src/streaming/models/CmsdModel.js

```diff
@@ -121,70 +110,33 @@ function CmsdModel() {
         })
     }
 
-    function _parseParameterValue(value) {
-        // If the value type is BOOLEAN and the value is TRUE, then the equals sign and the value are omitted
-        if (!value) {
-            return true;
-        }
-        // Check if boolean 'false'
-        if (value.toLowerCase() === 'false') {
-            return false;
-        }
-        // Check if a number
-        if (integerRegex.test(value)) {
-            return parseInt(value, 10);
-        }
-        // Value is a string, remove double quotes from string value
-        return value.replace(/["]+/g, '');
-    }
-
     function _parseCMSDStatic(value) {
         try {
-            const params = {};
-            const items = value.split(',');
-            for (let i = 0; i < items.length; i++) {
-                // <key>=<value>
-                const substrs = items[i].split('=');
-                const key = substrs[0];
-                const v = _parseParameterValue(substrs[1]);
-                params[key] = v;
-            }
-            return params;
+            return decodeCmsdStatic(value);
         } catch (e) {
             logger.error('Failed to parse CMSD-Static response header value:', e);
         }
     }
 
     function _parseCMSDDynamic(value) {
         try {
-            const params = {};
-            const entries = value.split(',');
-            // Consider only last CMSD-Dynamic entry
-            const entry = entries[entries.length - 1];
-            const items = entry.split(';');
-            // Server identifier as 1st item
-            for (let i = 1; i < items.length; i++) {
-                // <key>=<value>
-                const substrs = items[i].split('=');
-                const key = substrs[0];
-                const v = _parseParameterValue(substrs[1]);
-                params[key] = v;
-            }
-            return params;
+            const items = decodeCmsdDynamic(value);
+            const last = items[items.length - 1];
+            return last?.params || {};
         } catch (e) {
             logger.error('Failed to parse CMSD-Dynamic response header value:', e);
-            return [];
+            return {};
         }
     }
 
     function _mediaTypetoObjectType(mediaType) {
-        return MEDIATYPE_TO_OBJECTTYPE[mediaType] || OBJECT_TYPES.OTHER;
+        return MEDIATYPE_TO_OBJECTTYPE[mediaType] || CmsdObjectType.OTHER;
     }
 
     function _getParamValueForObjectType(paramsType, ot, key) {
         const params = paramsType === CMSD_STATIC ? _staticParamsDict : _dynamicParamsDict;
         const otParams = params[ot] || {};
-        const streamParams = params[OBJECT_TYPES.STREAM] || {};
+        const streamParams = params[CmsdObjectType.STREAM] || {};
```

Comment:
`CmsdObjectType.STREAM` seems to `undefined`. If we need `stream` in multiple places, please add a constant in this class. Ideally this can be added to the CommonMediaLibrary

Reply:
`"stream"` isn't defined in the CMSD spec for object type, so I've added it as a `const`.